Teensy/src/gpio/gpio.cpp

```diff
@@ -60,7 +84,60 @@ static int32_t gpio_getGearPosition()
 
 static uint32_t gpio_getSteeringWheelPosition()
 {
-    return 0;
+    #include <Wire.h>
+    #include <Arduino.h>
+   const int MPU = 0x68; //MPU6050 I2C Address
+   float AccX, AccY, AccZ;
+   float GyroX, GyroY, GyroZ;
+   float gyroAngleX, gyroAngleY, gyroAngleZ;
+   float elapsedTime, currentTime, previousTime;
+   float startPosX, startPosY, startPosZ;
+   float currentPosX, currentPosY, currentPosZ;
+   float yaw;
+   float calibrationvalue;
+   int c = 0;
+
+
+    Serial.begin(19200);
+    Wire.begin();
+    Wire.beginTransmission(MPU);
+    Wire.write(0x6B);
+    Wire.write(0x00);
+    Wire.endTransmission(true);
+    delay(20);
+
+
+    // Setting starting position of the sensor //
+
+    startPosX = 0;
+    startPosY = 0;
+    startPosZ = 0;
+
+    // Reading Gyroscope data //
+
+    previousTime = currentTime;
+    currentTime = millis();
+    elapsedTime = (currentTime - previousTime)/1000; //Outputs the elapsed time in seconds
+
+    Wire.beginTransmission(MPU);
+    Wire.write(0x43); //Starts transmission with Gyroscope output
+    Wire.endTransmission(false);
+    Wire.requestFrom(MPU,6,true);
+
+    calibrationvalue = 16.375; // Value is the raw value divided by the transformation to get to a range 2000 deg
```

Comment:
Needs a type definition, can also be defined as a const. Really an enum would be preferable here, but not necessary. So should be something like - 
```
const float32_t calibrationValue = 16.375F
```